guest/internal/prefilter/prefilter_exports.go

```diff
@@ -72,3 +84,129 @@ func _prefilter() uint32 { //nolint
 
 	return imports.StatusToCode(status)
 }
+
+// _addPod is only exported to the host.
```

Comment:
Why did you merge prefilter and prefilterextensions into one file? I might forget the context.

Reply:
@Gekko0114 
Sorry for late reply(I was too busy for work...). The context is below:
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/pull/92#discussion_r1536753108
I agreed that putting `addPod/removePod` in same place where the `_prefilter` is defined.